src/plugins/intel_cpu/src/nodes/memory.hpp

```diff
@@ -204,7 +222,15 @@ class MemoryInput : public MemoryInputBase {
 private:
     void assignStateHook() override {/*pass*/}
 
+    bool haveSubgraph() const {
+        return body != nullptr;
+    }
+
 private:
+    std::shared_ptr<ov::Model> body = nullptr;
+    ov::intel_cpu::Graph subGraph;
```

Comment:
I don't think so, because `body` is also used in other places, and getSubGraph() also return it to outside.
If we use unique_ptr, it cannot be copied or assigned.

Reply:
I'm asking not about the `body` but about the `subGraph`

Reply:
But I found cpu plugin only support to CPP 11, std::make_unique is enabled since CPP 14.
If I upgrade CPP 11 to 14, it will have a big impact.
So I use shared_ptr first.

Reply:
We already solved this problem:
https://github.com/openvinotoolkit/openvino/blob/239466ca5d5333af11ee750260215941737383b3/src/plugins/intel_cpu/src/utils/general_utils.h#L48-L55
So just call `make_unique`

Reply:
Done